.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
we do not need this

Reply:
This was something automatically added by vscode. Removed in my rebase.